db/migrate/20151108202718_add_email_confirm_column_to_members.rb

```diff
@@ -0,0 +1,6 @@
+class AddEmailConfirmColumnToMembers < ActiveRecord::Migration
+  def change
+    add_column :members, :email_confirmed, :boolean
+    add_column :members, :confirm_token, :string
+  end
```

Comment:
I think either we have `email_confirmed` and `email_confirmation_token` or `confirmed` and `confirmation_token`. I think it's just about email and not about the whole account, so I would choose the former.